mistry_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/skroutz/mistry/utils"
+)
+
+func TestPruneZombieBuilds(t *testing.T) {
+	project := "hanging-pending"
+	cmdOut, err := cliBuildJob("--project", project)
+	if err != nil {
+		t.Fatalf("Error output: %s, err: %v", cmdOut, err)
+	}
+	path := filepath.Join(cfg.BuildPath, project, "pending")
+	_, err = utils.RunCmd(curfs.Create(path + "/foo"))
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = utils.RunCmd(curfs.Create(path + "/bar"))
```

Comment:
Use `filepath.Join` to create the paths.

Reply:
fixed